lib/console_web/controllers/router/device_controller.ex

```diff
@@ -154,6 +154,16 @@ defmodule ConsoleWeb.Router.DeviceController do
         result =
           Ecto.Multi.new()
           |> Ecto.Multi.run(:event, fn _repo, _ ->
+            event = case event["data"]["req"]["body"] do
+              nil -> event
+              _ ->
+                if String.contains?(event["data"]["req"]["body"], <<0>>) or String.contains?(event["data"]["req"]["body"], <<1>>) or String.contains?(event["data"]["req"]["body"], "\\u0000\\") do
+                  Kernel.put_in(event["data"]["req"]["body"], "Unsupported unicode escape sequence in request body")
```

Comment:
would this break the frontend since "body" is usually an object and not a string for successful events

Reply:
ah, good point. I shall test further

Reply:
![image](https://user-images.githubusercontent.com/51131939/124013916-bf6fdb00-d997-11eb-98f7-1f75141cd10e.png)
all good on the front end